src/cfnlint/core.py

```diff
@@ -5,20 +5,30 @@
 import logging
 import os
 import sys
+
 from jsonschema.exceptions import ValidationError
 
-from cfnlint import Template
+from cfnlint.decorators.refactored import refactored
+import cfnlint.runner
+from cfnlint.template import Template as _Template
 from cfnlint.rules import RulesCollection
 import cfnlint.config
 import cfnlint.formatters
 import cfnlint.decode
 import cfnlint.maintenance
 from cfnlint.helpers import REGIONS
-
+from cfnlint.runner import Runner as _Runner
 
 LOGGER = logging.getLogger('cfnlint')
 DEFAULT_RULESDIR = os.path.join(os.path.dirname(__file__), 'rules')
 
+@refactored('Template is refactored and deprecated. Please use Template in cfnlint.template')
```

Comment:
Previously it was `from cfnlint import Template`. I think these should be moved to the `src/cfnlint/__init__.py` file instead.